Client.xcodeproj/xcshareddata/xcschemes/Fennec.xcscheme

```diff
@@ -102,6 +102,16 @@
             ReferencedContainer = "container:Client.xcodeproj">
          </BuildableReference>
       </MacroExpansion>
+      <CommandLineArguments>
+         <CommandLineArgument
+            argument = "-com.apple.CoreData.ConcurrencyDebug 1"
```

Comment:
This flag allows us to catch concurrency bugs, accessing contextes from wrong thread etc. I turned it on by default on development and test schemes.